components/imageGrid/index.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Masonry from 'react-masonry-css';
+import imageGridStyles from '@app/components/imageGrid/imageGrid.module.css';
+import PropTypes from 'prop-types';
+import { createStyles } from '@mantine/core';
+import cx from 'classnames';
+
+ImageGrid.propTypes = {
+  photos: PropTypes.arrayOf(PropTypes.string).isRequired,
+  numberOfColumns: PropTypes.number,
+  spacing: PropTypes.number,
+};
+
+ImageGrid.defaultProps = {
+  spacing: 10,
+};
+
+const defaultBreakpointColumns = {
+  default: 4,
+  1100: 3,
+  700: 2,
+  500: 1,
+};
+
+export default function ImageGrid({ photos, numberOfColumns, spacing }) {
+  const useStyles = createStyles(() => ({
+    columnSpacing: {
+      paddingLeft: `${spacing}px`,
+    },
+  }));
+  const { classes } = useStyles();
```

Comment:
little confused about this function, what's the difference between using this and imageGridStyles?